common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -269,6 +269,7 @@
   "feature.wynntils.quickCast.levelRequirementNotReached": "You need to reach %s %s before using this weapon.",
   "feature.wynntils.quickCast.name": "Quick Spell Cast Feature",
   "feature.wynntils.quickCast.notAWeapon": "You can only quick-cast with a weapon!",
+  "feature.wynntils.soulPointTimer.lore": "§bTime until next soul point: ",
```

```suggestion
  "feature.wynntils.soulPointTimer.lore": "Time until next soul point: %s",
```

This should support every language (where the last part is not last). Use this: `new TranslatableComponent(key, s1, s2)`